### DISCUSSION TITLE
Suggestion for Integrating HASH (HTTP Agnostic Software Honeypot) into T-Pot

### DISCUSSION THREAD

**Discussion by rubber-duck-br** [telekom-security/tpotce#1534] [Ideas]
I'm proposing the integration of the HASH project (HTTP Agnostic Software Honeypot), available on GitHub at https://github.com/DataDog/HASH, into T-Pot. HASH is a configurable honeypot that simulates various services to capture a wide range of attacks, enhancing threat detection and analysis. This integration could improve T-Pot’s capability to handle HTTP-based attacks and provide a more comprehensive threat mitigation tool.

Thanks you guys !

**Comment by t3chn0m4g3** [comment]
Already on the consideration list.
